DOCS-5584 preview and publish community (#99)
mulesoft/docs-api-community-manager: modules/ROOT/pages/preview-community.adoc

> endif::[]

Reviewer: NOTE
Very quick copyedit of changed material only

:imagesdir: ../assets/images/

> After you have edited your community site, you can preview the new version of the site to ensure that it meets your expectations:

Reviewer: ```suggestion
After you edit your community site, you can preview the new version to ensure that it meets your expectations:
```

> . From the API Community Manager (ACM) Control Panel, click *Community Builder*.

Reviewer: ```suggestion
. From the API Community Manager (ACM) control panel, click *Community Builder*.
```

. Click *Preview*.

> . Examine your changes and verify that they appear correctly. APIs that are only visible to administrators will appear in this preview.

Reviewer: ```suggestion
. Verify that your changes are applied and display correctly. APIs that are visible only to administrators appear in this preview.
```

Reviewer: ...but is "only" meant to modify "APIs": Only APIs that are visible to administrators...?

Author: Your change is exactly correct. APIs that are visible only to admins appear in this preview.

Author: And other APIs also appear in this preview.
